README.md

```diff
@@ -21,32 +29,121 @@ $ kubectl create secret docker-registry iterativeai \
+will be accessible from (e.g. `studio.iterative.ai` for the official SaaS
+installation). 
+
+> **Note**: We will now go over generating TLS cert with the `openssl` command.
+> If you already have a TLS certificate available, skip ahead to loading the
+> crt/key files into the kubernetes secret
+
+As an example, we'll assume your Studio installation will be available
+via: `https://my-studio.private.com`. We'll also use a self-signed
+certificate for the sake of simplicity. Of course, for a production
+installation we recommend using a certificate signed by a trusted CA.
+
+Let's create a self-signed SSL cert using `openssl` (you may need to install it):
+```bash
+openssl req \
+  -x509 -newkey rsa:4096 -sha256 -nodes \
+  -keyout tls.key -out tls.crt \
+  -subj "/CN=my-studio.private.com" \
+  -addext "subjectAltName = DNS:my-studio.private.com" \
+  -days 365
+```
+
+This will create the files `tls.crt` & `tls.key` in your current dir.
+
+Now, let's create a TLS secret containing the contents from this cert:
+
+```bash
+kubectl create secret tls studio-ingress-tls \
+  --namespace studio \
+  --cert=tls.crt \
+  --key=tls.key
 ```
 
-Deploy Studio
+We will refer to this secret in the below installation instructions.
+
+### Install Studio
 
+Now, we are ready to deploy Studio using the Helm chart.
+
+Add the iterative Helm repository:
 ```bash
 $ helm repo add iterative https://helm.iterative.ai
-$ helm install studio iterative/studio --namespace studio -f override.yaml
 ```
 
-## Update Studio
+### Minimal Installation:
+
+To install studio with all default values (for sanity, testing), 
+Run the following command:
+```bash
+$ helm install studio iterative/studio \
+    --namespace studio \
+    --set-json='imagePullSecrets=[{"name": "iterativeai"}]'
+```
+
+### Functional Installation:
 
-Studio's `values.yaml` file points to the `latest` image tag, instructing HELM to always pull down the latest image from the registry. 
+Realistically, for a functional Studio app instance, you'll
+need to configure multiple values. In this example we'll prepare a
+more realistic and functional installation.
 
-**Optional Step** 
+Assumptions:
+- We have an ingress controller (nginx) installed on the cluster.
+- Studio will be available from the domain: `my-studio.private.com`
+  (we've registered this domain and made sure to redirect to our cluster).
+- We refer to the TLS secret `studio-ingress-tls` we've created in [a previous step](#prepare-tls-secret).
 
-If the requirement is to install/update Studio to a specific version, update the `override.yaml` file with the following:
+Create a file named `studio-values.yaml`, with the following contents:
 
 ```yaml
 imagePullSecrets:
   - name: iterativeai
 
+global:
+  host: "my-studio.private.com"
+  ingress:
+    enabled: true
+    className: nginx
+    tlsEnabled: true
+    tlsSecretName: studio-ingress-tls
+  blobvault:
+    accessKeyId: <aws-access-key-id>
+    secretAccessKeyId: <aws-secret-access-key>
```

Comment:
We're already configuring blobvault by default via the minio dependency. 
This should be optional if the user wants to use cloud S3 instead. I think we can leave it out for now.

Reply:
removed